MediaBrowser.WebDashboard/dashboard-ui/wizardfinish.html

```diff
@@ -9,38 +9,11 @@ <h1>
 
             <p style="margin:2em 0;">${WizardCompleted}</p>
 
-            <p style="margin:2em 0;" class="appLinks">
-                <a href="https://www.amazon.com/Emby-Mobile/dp/B00GVH9O0I" target="_blank">
-                    <img src="https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/amazon.png" title="Android via Amazon App Store" />
-                </a>
-                <a href="https://play.google.com/store/apps/details?id=com.mb.android&hl=en" target="_blank">
-                    <img src="https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/playstore.png" title="Android via Google Play Store" />
-                </a>
-                <a href="https://itunes.apple.com/us/app/emby/id992180193?ls=1&mt=8" target="_blank">
-                    <img src="https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/ios.png" title="iOS" />
-                </a>
-                <a href="https://channelstore.roku.com/details/44191/emby" target="_blank">
-                    <img src="https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/roku.jpg" title="Roku" />
-                </a>
-                <a href="http://www.windowsphone.com/s?appid=f4971ed9-f651-4bf6-84bb-94fd98613b86" target="_blank">
-                    <img src="https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/windowsphone.png" title="Windows Phone" />
-                </a>
-                <a href="http://apps.microsoft.com/windows/en-us/app/media-browser/ad55a2f0-9897-47bd-8944-bed3aefd5d06" target="_blank">
-                    <img src="https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/windowsrt.png" title="Windows 8.1" />
-                </a>
-                <a href="https://emby.media/emby-for-windows-media-center.html" target="_blank" title="Windows Media Center">
-                    <img src="https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/mbc.png" />
-                </a>
-                <a href="https://emby.media/emby-for-kodi.html" target="_blank" title="Kodi">
```

Comment:
Just a thought - Jellyfin for Kodi might be something worth keeping eventually...

Reply:
Agreed, we shouldn't strip out things that help us, like 3rd party integrations. BUT on the same token I don't want to link to them, hard problem. I'd err on "remove from the wizard" for now though.

Reply:
That was just a stray thought, not completely related to this particular wizard removal.